circle.yml

```diff
@@ -5,7 +5,6 @@ machine:
   - docker
 test:
   override:
-  - bundle install
```

Comment:
I think we'll want to keep `bundle install` to install the dev dependencies specified in the gemspec.